packages/kit/src/runtime/app/state/client.js

```diff
@@ -0,0 +1,29 @@
+import { page as _page } from '../../client/state.svelte.js';
+
+/** @type {import('@sveltejs/kit').Page} */
+export const page = {
+	get data() {
+		return _page.data;
```

Comment:
Wouldn't it make sense to also use context on the client? I'm thinking of shallow-routing in a modal where we might want to have a different `page.data`, `page.url`, etc. in the modal than in the background page.

Reply:
No, that's not what `page` is. It's global. And it would be confusing if `$app/state` was different than `$app/stores` beyond using state instead of stores